### DISCUSSION TITLE
Implement Live collaboration on top of <Excalidraw />

### DISCUSSION THREAD

**Discussion by alerodriguez01** [excalidraw/excalidraw#7266] [Q&A]
Hello team,

I'm currently integrating `<Excalidraw />` into my project, and its UI customization capabilities aligns perfectly with our requirements. However, i'm facing difficulties incorporating the collaboration mode, a crucial feature for our project as it revolves around a collaborative learning platform.

I looked up the [*Collaboration mode - Self-hosting vs Collab on top of `<Excalidraw />`*](https://github.com/excalidraw/excalidraw/discussions/3879) #3879 discussion and the [answer](https://github.com/excalidraw/excalidraw/discussions/3879#discussioncomment-1110524) by Aakansha and reproduced the following steps:
- Clone [excalidraw-room](https://github.com/excalidraw/excalidraw-room)  and host the server myself 
- Clone [excalidraw](https://github.com/excalidraw/excalidraw)
  -  Update the [entry](https://github.com/excalidraw/excalidraw/blob/master/src/packages/excalidraw/webpack.prod.config.js#L12) in *webpack.prod.config.js* to ["../../../excalidraw-app/index.tsx"](https://github.com/excalidraw/excalidraw/blob/master/excalidraw-app/index.tsx)
- Release the npm package: execute `yarn build:umd` and then `yarn publish` 
 
I have no problems through these steps, but when i try to import the package and use the [`<ExcalidrawApp />`](https://github.com/excalidraw/excalidraw/blob/master/excalidraw-app/index.tsx#L801), it turns out that it can't be found.

Am i overlooking something or is there a step I might have missed?

Your guidance on this matter would be greatly appreciated.

Thank you.

**Comment by dakosek** [comment]
I am looking at KovaJS or doing what you’re doing here.  Have you added any custom components or templates yet? or is that what you were repairing to do when you ran into this issue?
